utils/collect-updates/__tests__/lib-collect-dependents.test.js

```diff
@@ -65,5 +66,9 @@ test("internal node (cycle)", () => {
 
   const result = collectDependents(candidates);
 
-  expect(Array.from(result)).toEqual([expect.objectContaining({ name: "package-cycle-1" })]);
+  expect(Array.from(result)).toEqual([
+    expect.objectContaining({ name: "package-cycle-1" }),
+    expect.objectContaining({ name: "package-cycle-extraneous-2" }),
+    // package-cycle-extraneous-1 was ignored
```

Comment:
Please excuse me on commenting on old PR, but what is the reason behind ignoring `package-cycle-extraneous-1`? I know it is directly because `siblingDependents.has(currentNode.name)` evaluates to `true`, but what is the intention behind this? Thanks